ui/jobs/bar.d.ts

```diff
@@ -0,0 +1,61 @@
+import { Job } from '../../types/job';
+import { JobDetail } from '../../types/event';
+import { ComboCallback } from './combo_tracker';
+
+export interface TimerBar {
+  duration: number;
+}
+
+export interface Box extends HTMLElement {
+  elapsed: number;
+  duration: number;
+  threshold: number;
+  valuescale: number;
+  fg: string;
+  // parentNode: { classList: ()};
+}
+
+type Selector = {
+  id?: string;
+  fgColor?: string;
+  classList?: string[];
+  threshold?: number;
+};
+
+interface ResourceBox extends HTMLDivElement {
+  parentNode: {
+    classList: {
+      add: (s: string) => void;
+      remove: (s: string) => void;
+    };
+  };
+}
+
+export interface Bars {
```

Comment:
Would it be better to just export `Bars` directly from jobs rather than redefining it here? Or is this just temporary until you convert `Bars`? (If it is temporary, maybe a `TODO: remove this when jobs.js is converted to typescript?` sort of thing?)

Reply:
it's temporary.

Reply:
If it is temporary, could you add a `TODO: remove this when jobs.js is converted to typescript` line?